packages/algorand-js/src/lib/parsing.ts

```diff
@@ -86,7 +94,7 @@ export function convertToBuffer (s: string, encoding?: EncodingType): Buffer {
 }
 
 /**
- * Description: Returns base64 or base32 string
+ * Returns base64 or base32 string
```

Comment:
Please update the the description about the `args` argument and describe when we convert to base64 and when to base32

Reply:
ok